packages/svelte/package.json

```
@@ -2,9 +2,9 @@
 	"name": "@carbon/charts-svelte",
 	"version": "0.54.0",
 	"description": "Carbon charting components for Svelte",
-	"main": "./bundle.js",
-	"module": "./index.js",
-	"svelte": "./src/index.js",
+	"type": "module",
+	"main": "./index.js",
```

When building the `svelte` charts, the `src` is copied to the dist folder, so the Svelte/ESM entry should be "./src/index.js".

```suggestion
	"main": "./src/index.js",
```